examples/example_plugin/example_plugin/models.py

```diff
@@ -28,12 +29,16 @@ def get_absolute_url(self):
         return reverse("plugins:example_plugin:examplemodel", kwargs={"pk": self.pk})
 
     def to_objectchange(self, action):
+        serializer_class = get_serializer_for_model(self.__class__)
+        serialized_data = serializer_class(self, context={"request": None}).data
+        object_datav2 = convert_set_to_list_in_obj(serialized_data)
```

Comment:
I wouldn't have thought that serializer data would contain any `set` values. What was the case you encountered where this cleanup is necessary?

Reply:
Yes

Reply:
... What was the case where this happened?

Reply:
Encounter it while running test` nautobot.extras.tests.test_api.GitRepositoryTest.test_bulk_create_objects`

Reply:
OK, so it appears that `MultipleChoiceField` gets rendered into `serializer_class().data` as a `set` object. Makes sense, but interesting that DjangoJSONEncoder doesn't handle the `set` data type natively. We do have the `nautobot.core.celery.NautobotKombuJSONEncoder` class that *does* handle sets, so it might be that a more straightforward fix here would be to use that class as the encoder for the `ObjectData.object_datav2` field so that it automatically handles sets?